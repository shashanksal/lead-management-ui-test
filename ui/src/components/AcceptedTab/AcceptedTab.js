import React, { useState } from "react";

//Material UI Components
import Box from "@mui/material/Box";
import Typography from "@mui/material/Typography";
import Grid from "@mui/material/Grid";

// Custom components import
import AcceptedJobCard from "./AcceptedJobCard/AcceptedJobCard";

const AcceptedTab = () => {
	return (
		<Box>
			<AcceptedJobCard />
		</Box>
	);
};

export default AcceptedTab;
