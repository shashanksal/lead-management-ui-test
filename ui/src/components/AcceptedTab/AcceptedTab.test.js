import React from "react";
import { render } from "@testing-library/react";

import AcceptedTab from "./AcceptedTab";

describe("AcceptedTab", () => {
	it("should match snapshot", () => {
		const { asFragment } = render(<AcceptedTab />);
		expect(asFragment()).toMatchSnapshot();
	});
});
