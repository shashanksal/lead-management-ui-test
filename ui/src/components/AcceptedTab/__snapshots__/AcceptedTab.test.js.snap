// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AcceptedTab should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
    >
      No accepted jobs to show
    </div>
  </div>
</DocumentFragment>
`;
